OpenIPSL/Electrical/Solar/PowerFactory/DIgSILENT/Auxiliary/ActivePowerController.mo

```diff
@@ -0,0 +1,114 @@
+within OpenIPSL.Electrical.Solar.PowerFactory.DIgSILENT.Auxiliary;
+
+model ActivePowerController
```

Comment:
Please add a short descriptive string to this and also to all the other models which are added.

Reply:
Done in fa63456.